app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h2>
+  Edit Task
+</h2>
+<%= form_with model: @task do |f| %>
+<p>Fill out the following to save a task:</p>
+<%= f.label :action %>
+<%= f.text_field :action  %>
```

Comment:
You've got almost exactly the same code here to render a form as you do in `new`. You should dry this up with a view partial.